qchem/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Release 0.10.0 (development release)
+# Release 0.10.0 (current release)
```

Comment:
@agran2018, can you please check the changelog for QChem, and let me know if it all looks okay 🙂